css-backgrounds-4/Overview.bs

```diff
@@ -289,15 +289,14 @@ The 'border-block-start-radius', 'border-block-end-radius',
 		Animatable: see individual properties
 	</pre>
 
-	<p>These properties correspond to the 'border-top-radius',
-	'border-right-radius', 'border-bottom-radius', and 'border-left-radius'
-	shorthand properties.
-	The mapping depends on the element’s 'writing-mode', 'direction',
-	and 'text-orientation',
-	with start/end referring to the axis orthogonal to the one given
-	by the first part of the name
-	(i.e. patterned as 'border-block-<i>inline</i>-radius' and
-	'border-inline-<i>block</i>-radius')
+	<p>These properties correspond to the 'border-start-start-radius',
+	'border-start-end-radius', 'border-end-start-radius',
+	and 'border-end-end-radius'	longhand properties.
+	The start/end refers to the axis given by the first part of the name
+	(i.e. 'border-start-<i>inline</i>-radius' for 'border-block-start-radius`,
+	'border-end-<i>inline</i>-radius' for 'border-block-end-radius',
+	'border-<i>block</i>-start-radius' for 'border-inline-start-radius', and
+	'border-<i>block</i>-end-radius' for 'border-inline-end-radius')
```

Comment:
I basically copied the wording from https://drafts.csswg.org/css-logical-1/#border-radius-properties. There, the italic written parts are meant to clarify the logical direction and stand for `start` and `end`.

The sentence is meant to say that `border-block-start-radius` is the shorthand for `border-start-start-radius` and `border-end-start-radius`, `border-block-end-radius` the shorthand for `border-end-start-radius` and `border-end-end-radius`, `border-inline-start-radius` the shorthand for `border-start-start-radius` and `border-end-start-radius`, and `border-inline-end-radius` the shorthand for `border-start-end-radius` and `border-end-end-radius`.
Though that is obviously even longer and quite complicated.
So maybe there's an easier way to express that?

(Also, the order of the longhands is meant to follow the ones of the physical shorthands, so it is always `start` first then `end`.)

Reply:
I find it confusing because the wording from https://drafts.csswg.org/css-logical-1/#border-radius-properties explains the mapping of logical longhands into physical longhands, but here it's trying to explain how some shorthands expand into longhands.

I would rather copy the wording from the physical shorthands

> The physical 'border-\*-radius' shorthands set the two physical 'border-\*-\*-radius'
> longhand properties of the related physical side. [...]
> The two values for the radii are given in the order
> top-left, top-right for 'border-top-radius',
> top-right, bottom-right for 'border-right-radius'
> bottom-left, bottom-right for 'border-bottom-radius',
> and top-left, bottom-left for 'border-left-radius'.

> The logical 'border-\*-radius' shorthands set the two logical 'border-\*-\*-radius'
> longhand properties of the related logical side. [...]
> The two values for the radii are given in the order
> start-start, start-end for 'border-block-start-radius',
> end-start, end-end for 'border-block-end-radius'
> start-start, end-start for 'border-inline-start-radius',
> and start-end, end-end for 'border-inline-end-radius'.

Or maybe merge the sections and define all the shorthands together.

> The 'border-\*-radius' shorthands set the two 'border-\*-\*-radius'
> longhand properties of the related side. [...]
> The two values for the radii are given in the order
> top-left, top-right for 'border-top-radius',
> top-right, bottom-right for 'border-right-radius'
> bottom-left, bottom-right for 'border-bottom-radius',
> top-left, bottom-left for 'border-left-radius',
> start-start, start-end for 'border-block-start-radius',
> end-start, end-end for 'border-block-end-radius'
> start-start, end-start for 'border-inline-start-radius',
> and start-end, end-end for 'border-inline-end-radius'.

Reply:
Sounds reasonable. I've merged the two sections now.